Code/GraphMol/Wrap/rough_test.py

```diff
@@ -2703,6 +2703,64 @@ def test65StreamSupplier(self):
       i += 1
     self.assertEqual(i, 16)
 
+  def testMaeStreamSupplier(self):
+    try:
+        MaeMolSupplier = Chem.MaeMolSupplier
+    except AttributeError:  # Built without Maestro support, return w/o testing
+        return
+
+    fileN = os.path.join(RDConfig.RDBaseDir, 'Code', 'GraphMol', 'FileParsers', 'test_data',
+                         'NCI_aids_few.maegz')
+    molNames = ["48", "78", "128", "163", "164", "170", "180", "186", "192", "203", "210", "211",
+                "213", "220", "229", "256"]
+    inf = gzip.open(fileN)
+    if 0:
```

Comment:
There's not really any need to duplicate this leftover from previous times.
Skipping the `if 0` probably makes sense.

Reply:
Done